pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -51,7 +51,9 @@ func NewREST() *REST {
 }
 
 func (r *REST) New() runtime.Object {
-	return &statsv1alpha1.NodeLatencyStats{}
+	return &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta: metav1.ObjectMeta{CreationTimestamp: metav1.Now()},
+	}
```

Comment:
I don't think this change is needed or even makes sense. I would not modify `New` as part of this PR.

Reply:
Yes, worked on it.